cdap-master/src/main/java/co/cask/cdap/data/tools/DatasetUpgrader.java

```diff
@@ -52,6 +54,7 @@
   private final QueueAdmin queueAdmin;
   private final HBaseTableUtil hBaseTableUtil;
   private final DatasetFramework namespacedFramework;
+  private static final Pattern USER_TABLE_PREFIX = Pattern.compile("cdap\\.user\\..*");
```

Comment:
Yes, user tables until 2.7 always had the ``cdap.user.`` prefix. However, I think if there is a way to to indicate starts with (^ maybe), we should do that, to be doubly sure.